File: templates/_composition/CodeBehind/Project.SplitView.AddToPrimaryNavigationItemsVB/Views/ShellPage.xaml_postaction.vb
'{**
'This code block adds the wts.ItemNamePage to the _primaryItems of the ShellPage.
'**}

Namespace Views
    Public NotInheritable Partial Class ShellPage
        Inherits Page
        Implements INotifyPropertyChanged

        Private Sub PopulateNavItems()
            '^^
            '{[{
            _primaryItems.Add(ShellNavigationItem.FromType(Of wts.ItemNamePage)("Shell_wts.ItemName".GetLocalized(), Symbol.Document))
            '}]}
        End Sub
    End Class
End Namespace

File: templates/Uwp/_composition/CodeBehind/Project.SplitView.AddToPrimaryNavigationItems/Views/ShellPage.xaml_postaction.cs
//{**
//This code block adds the wts.ItemNamePage to the _primaryItems of the ShellPage.
//**}

namespace Param_ItemNamespace.Views
{
    public sealed partial class ShellPage : Page, INotifyPropertyChanged
    {
        private void PopulateNavItems()
        {
            //^^
            //{[{
            _primaryItems.Add(ShellNavigationItem.FromType<wts.ItemNamePage>("Shell_wts.ItemName".GetLocalized(), Symbol.Document));
            //}]}
        }
    }
}
